backend/routes/image_generation/image_generation_api.js

```diff
@@ -259,6 +259,7 @@ router.post("/generate", [
 
     res.status(202).json({ success: true, jobId });
   } catch (e) {
+    errorRetries += 1;
```

Comment:
This is still broken. errorRetries is not defined, nor would it's state persist anyways. Disable the max retries logic on backend please

Reply:
Thanks for letting me know. 👍